editor/src/components/canvas/canvas.ts

```diff
@@ -137,21 +137,10 @@ const Canvas = {
       }
     }
 
-    const storyboardComponent = metadata.components.find((c) =>
-      TP.pathsEqual(c.scenePath, EmptyScenePathForStoryboard),
-    )
-
-    if (storyboardComponent != null) {
-      const storyboardEntryPoint = MetadataUtils.getImmediateChildren(
-        metadata,
-        storyboardComponent.scenePath,
-      )
-      return storyboardEntryPoint.flatMap((storyboardRoot) => {
-        return recurseChildren({ x: 0, y: 0 } as CanvasVector, storyboardRoot).frames
-      })
-    } else {
-      return []
-    }
+    const storyboardEntryPoint = MetadataUtils.getAllStoryboardChildren(metadata)
```

Comment:
this could be `canvasRoots` or `storyboardRoots` or `storyboardChildren`

Reply:
`storyboardChildren` sounds right to me

Reply:
(I'm going to roll this change into the next pass, since I'm working on that now)